mlflow/projects/__init__.py

```diff
@@ -299,16 +287,14 @@ def _maybe_set_run_terminated(active_run, status):
         active_run.set_terminated(status)
 
 
-def _get_entry_point_command(project_dir, entry_point, use_conda, parameters, storage_dir):
-    project = _load_project(project_dir=project_dir)
+def _get_entry_point_command(project, entry_point, use_conda, parameters, storage_dir):
     storage_dir_for_run = _get_storage_dir(storage_dir)
     eprint("=== Created directory %s for downloading remote URIs passed to arguments of "
            "type 'path' ===" % storage_dir_for_run)
     commands = []
     if use_conda:
-        conda_env_path = os.path.abspath(os.path.join(project_dir, project.conda_env))
         activate_path = _get_conda_bin_executable("activate")
-        commands.append("source %s %s" % (activate_path, _get_conda_env_name(conda_env_path)))
+        commands.append("source %s %s" % (activate_path, _get_conda_env_name(project)))
```

Comment:
Is there any unit test which confirms that we can successfully run through this code with an absent conda env, but still create and activate the empty conda environment? (That is the behavior we expect, right?)